inject/src/main/java/io/micronaut/context/env/PropertySourcePropertyResolver.java

```diff
@@ -909,28 +907,6 @@ public void close() throws Exception {
         }
     }
 
-    /**
-     * Return logEnabled value.
-     *
-     * @return is log enabled
-     *
-     * @since 3.9.0
-     */
-    public boolean isLogEnabled() {
-        return logEnabled;
-    }
-
-    /**
-     * Setter for logEnabled.
-     *
-     * @param logEnabled is log enabled
-     *
-     * @since 3.9.0
-     */
-    public void setLogEnabled(boolean logEnabled) {
-        this.logEnabled = logEnabled;
-    }
-
```

Comment:
removing these methods is a breaking change

Reply:
Reverted these method with mark Deprecated